servers/audio_server.h

```diff
@@ -56,15 +56,17 @@ class AudioDriver {
 	SafeNumeric<uint64_t> prof_time;
 #endif
 
+	_FORCE_INLINE_ static float _read_sample(const void *p_buffer, int p_idx);
+
 protected:
-	Vector<int32_t> input_buffer;
+	Vector<AudioFrame> input_buffer;
 	unsigned int input_position = 0;
 	unsigned int input_size = 0;
 
-	void audio_server_process(int p_frames, int32_t *p_buffer, bool p_update_mix_time = true);
+	void audio_server_process(int p_frames, void *p_buffer, bool p_active = true, bool p_update_mix_time = true);
```

Comment:
Not really sure about `p_active`, it tells whether or not to process audio or just fill buffer with 0. I think it should be an argument, because `AudioDriverOpenSL` and `AudioDriverCoreAudio` pass false if callback is unable to lock mutex, it doesn't feel right to have a function like `is_active` because of that. `input_process` has something simmilar to this, if you pass `nullptr` as a buffer (it's needed for `AudioDriverWASAPI`), internall buffer will be filled with 0, we can't pass `nullptr` in `audio_server_process`, as we still need to fill a buffer. It's just a simplification to not call `memset` and to not reapeat the same code